models/offer.js

```diff
@@ -41,6 +41,16 @@ const offerSchema = new Schema(
       },
       required: [true, 'Author role must be selected.']
     },
+    authorName: {
```

Comment:
can't we populate author name from userId field (will be authorId after Vladislav's changes)?

Reply:
I agree that it would be more logical, but as it turned out, there is no way to make populate() filter offers based on properties of the offer's author.